packages/databricks-vscode/DATABRICKS.quickstart.md

```diff
@@ -10,95 +6,117 @@ The Databricks extension for VS Code allows you to develop for the Databricks La
 
 # Features
 
+-   Create [Databricks Asset Bundle](https://docs.databricks.com/en/dev-tools/bundles/index.html)
+-   Manage, run and deploy Databricks assets (jobs, DLT Pipelines)
 -   Synchronize code to a Databricks workspace
 -   Run Python files on a Databricks cluster
 -   Run notebooks and Python files as Workflows
+-   Debug Spark code running on a Databricks cluster using Databricks Connect
+-   Run notebooks cell by cell using Databricks Connect
 
 ## <a id="toc"></a>Table of Contents
 
 -   [Getting Started](#setup-steps)
-    -   [Configure Extension](#configure-extension)
+    -   [Create a Databricks Project](#create-databricks-project)
     -   [Running Code](#running-code)
         -   [Running PySpark Code](#running-pyspark-code)
         -   [Running PySpark Code and Notebooks as Workflows](#running-code-as-workflows)
-        -   [Advanced: Running using custom run configurations](#run-configurations)
--   [Extension Settings](#settings)
--   [`Databricks:` Commands](#commands)
+    -   [Deploying Databricks Asset Bundle](#dabs)
```

Comment:
Similar to above, maybe let's make it plural here?

Reply:
Yes. (See above.) Since DABs is already spelled out above, it would also be OK to just use "bundle" here.